native_client/client.js

```diff
@@ -0,0 +1,14 @@
+const Fs = require('fs');
+const Sox = require('sox-stream');
+const Ds = require('./javascript/build/Release/deepspeech');
```

Comment:
How would this look like after installation of the module? Ideally this should be copy-pasteable into a file that doesn't live in this location.